app/api/v1/items/[id]/route.ts

```diff
@@ -1,47 +1,75 @@
 import { db } from '@/db/drizzle/db';
-
-// Uncomment if using `select` for querying
-// import { item } from '@/db/drizzle/schema/item';
-// import { eq } from 'drizzle-orm';
+import { item } from '@/db/drizzle/schema/item';
+import { format } from 'date-fns';
+import { eq } from 'drizzle-orm';
+import type { ItemProps } from '../definitions';
 
 export const dynamic = 'force-dynamic';
 
-export async function GET(request: Request, { params }: { params: { id: number } }) {
-  const id = parseInt(params.id);
-
-  // TODO: consider moving queries into a library
+//FETCH ITEM FROM DB BY ID
+export async function GET(
+  request: Request,
+  { params }: { params: { id: number } },
+) {
+  const id = params.id;
 
-  // Query using select
-  // const result = await db.select().from(item).where(eq(item.id, id));
-
-  // Query using findFirst
   const result = await db.query.item.findFirst({
     where: (item, { eq }) => eq(item.id, id),
     with: {
-      // Include relation
-      // with: {
-      //   auditHistory: true,
-      // }
-
-      // Include nested relations
-      auditHistory: {
+      auditHistories: {
         with: {
-          // Select all
-          // user: true,
-
-          // Exclude column
-          user: {
-            columns: {
-              password: false,
-            }
-          },
+          user: true,
```

Comment:
Can we specify returned columns, or at least exclude the password?

Reply:
we can do that, what specific columns you want to include and exclude